apps/mdn/interactive-examples/tf/mdninteractive.tf

```diff
@@ -91,18 +93,22 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
 
   logging_config {
     include_cookies = false
-    bucket          = "mdninteractive-logs.s3.amazonaws.com"
+    bucket          = "${local.interactive-bucket-logs}.s3.amazonaws.com"
     prefix          = "cflogs"
   }
 
-  aliases = ["interactive-examples.mdn.mozilla.net",
-             "interactive-examples.mdn.moz.works"]
+  aliases = [
+    # FIXME: Can't use mdn.mozilla.net because its already taken
+    # assuming its from whats running in prod right now
+    #"interactive-examples.mdn.mozilla.net",
+    "interactive-examples.mdn.mozit.cloud",
```

Comment:
@jwhitlock are you ok with the public domain being `interactive-examples.mdn.mozit.cloud`?

Another option would be `interactiveexamples.mdn.mozilla.net` (without the -)

Reply:
Just fyi its commented out for now, I can't think of another domain for it. But yeah picking another name would be something we need to do here just to get something going. Somewhere down the line when the cut over happens we can reuse `interactive-examples.mdn.mozilla.net` once the teardown on mozmeaos part is done

Reply:
we try to stick with "non weird" domains in public facing apps. `mozilla.com/org` is pretty restrictive, so we tend to use `mozilla.net` subdomains.

Reply:
> Maybe "FIXME: public name is owned by MozMEAO AWS account"?

I can fix that wording

> Is it possible to start that way, or will AWS prevent having two certs with the same alias?

Its not a cert problem, ACM I believe will happily create the cert for you. This is an issue because the CNAME `interactive-examples.mdn.mozilla.net` is already taken by another cloudfront distribution. I did find [this](https://aws.amazon.com/premiumsupport/knowledge-center/resolve-cnamealreadyexists-error/) which does seem to indicate that I can create it if I jump through some hoops, unsure if it will affect whats in prod now though